packages/internal-scripts/src/release-image.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import parseChangelog from 'changelog-parser';
+import commandLineArgs from 'command-line-args';
+import {marked} from 'marked';
+import puppeteer from 'puppeteer';
+import {readFile} from 'fs/promises';
+import path from 'path';
+
+const optionDefinitions = [
+  {name: 'file', type: String, defaultOption: true},
+  {name: 'version', type: String},
+];
+
+export const run = async () => {
+  const options = commandLineArgs(optionDefinitions);
+
+  if (!options.filename && !options.version) {
```

Comment:
This check always fails. It should check `options.file`, if that is meant to be required. It does mean the defaulting to `CHANGELOG.md` in line 44 no longer really makes sense, which I think is fine since our top level changelog file isn't really being updated anyway so I'd say it's okay to require the file input.

Also, `options.version` doesn't need to be checked since it'll default to latest version if not specified.

Reply:
Whoops! Great catch!